test/index.js

```diff
@@ -560,4 +568,33 @@ describe('Marked renderer', () => {
       '<img src="http://bar.com/b.jpg" alt="a&quot;b" title="c&gt;d"></p>\n'
     ].join('\n'));
   });
+
+  describe('exec filter to extend', () => {
+    it('should render image url to data-src', () => {
```

Comment:
```suggestion
    it('should execute filter registered to marked:renderer', () => {
```

`render image url to data-src` is not a part of `hexo-renderer-marked`'s feature. The filter `marked:renderer` is.